### ISSUE TITLE
How to reduce iterations of fitting model?

### ISSUE THREAD

**Issue by WilliamQue** [facebook/prophet#1863]
By default, prophet will stop until iterations reaching 10000, Is there parameters to reduce it, or a way to set another convergence rule, for example, iteration will be stopped when log joint probability improving less than 0.01?

**Comment by bletham** [comment]
Yes, if you pass along a kwarg `iter` to `m.fit(df, iter=1e3)` then it will use that many iterations.

Any kwarg that you put there will be passed along to pystan, and you can see what the available kwargs are by looking at
```Python
import pystan
help(pystan.StanModel.optimizing)
```
It looks like there are some args there related to the optimizer tolerance, but that's all controlled on the Stan side.
